src/Marten.Testing/Bugs/Bug_1155_null_duplicate_fields.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System.Linq;
+using Marten.Services;
+using Xunit;
+
+namespace Marten.Testing.Bugs
+{
+    public class Bug_1155_null_duplicate_fields : IntegratedFixture
+    {
+        [Fact]
+        public void when_enum_is_null_due_to_nullable_type()
+        {
+            StoreOptions(_ =>
+            {
+                _.Serializer(new JsonNetSerializer {EnumStorage = EnumStorage.AsInteger});
+                _.Schema.For<Target>().Duplicate(t => t.NullableColor);
+            });
+
+            using (var session = theStore.OpenSession())
+            {
+                session.Store(new Target
+                {
+                    Number = 1,
+                    NullableColor = null
+                });
+
+                session.SaveChanges();
+            }
+
+            using (var session = theStore.OpenSession())
```

Comment:
As all tests have the same Asert logic - imho it would be good to move it to the private method and reuse it.